rosdep/python.yaml

```diff
@@ -1002,6 +1002,21 @@ python-httplib2:
     saucy: [python-httplib2]
     trusty: [python-httplib2]
     trusty_python3: [python3-httplib2]
+python-hypothesis:
+  debian:
+    sid: [python-hypothesis]
```

Comment:
Please remove `sid`. Since it is a rolling release we don't specify it explicitly. Since all keys on Debian are the same you could also make a generic entry (rather than making it distro specific): `debian: [python-hypothesis]`